woocommerce-myparcel.php

```diff
@@ -98,6 +98,16 @@ private function define($name, $value)
             }
         }
 
+        /**
+         * This method is used internally, to be able to use the staging environment of MyParcel
+         */
+        private function useStagingEnvironment(): void
+        {
+            if (get_option('use_myparcel_staging_environment')) {
+                putenv('API_BASE_URL=' . get_option('myparcel_base_url'));
```

Comment:
Ik ben niet voor environment gebruiken hiervoor, maar dat is uiteindelijk ook niet heel relevant voor deze story